scipy/special/_basic.py

```diff
@@ -0,0 +1,2307 @@
+#
```

Comment:
On GitHub it looks like you created a new file `_basic.py` and then copy-pasted all code, rather than doing `git mv`. Is that right? If so, `git mv` is preferable because it keeps the history / `blame` intact.

Reply:
Git does not track file renames, so there's no difference between the two operations from its point of view. It has heuristics to do the move tracking, but maybe github web interface doesn't have as good.

Reply:
Looking at individual commits I see it was a `git mv` then `git add` so I think the `blame` should be okay in the end.